tests/test_core.py

```diff
@@ -3301,7 +3284,6 @@ def test_dlfcn_longjmp(self):
         printf("pre %d\n", i);
       }
       ''')
-    self.set_setting('EXPORTED_FUNCTIONS', ['_jumpy'])
```

Comment:
We'd need this if we used mode 2, I think? Looks like we don't use that for these tests, so this should be an ok change, but maybe it's worth keeping for future use if we do want mode 2 here?

Reply:
Sure.. if we ever wanted to use `SIDE_MODULE=2` then we would need `EXPORTED_FUNCTIONS`.  As it stands I think this does nothing..

Using `SIDE_MODULE=2` would imply there is stuff in the side module that we don't need.. which is almost never the case in test code.

Reply:
I'd rather not keep settings around that are not used.